### PR TITLE
Add sections for dataset update and get

### PR DESCRIPTION
**Pull request by marianmosley**
For: 
https://github.com/mode/issues/issues/16192
https://github.com/mode/issues/issues/16188

### CODE REVIEW

**Comment by marianmosley** [comment]
For: 
https://github.com/mode/issues/issues/16192
https://github.com/mode/issues/issues/16188

File: swagger.yaml [hunk 1]
```diff
@@ -231,6 +231,70 @@ paths:
...
+            "$ref": "#/definitions/Unauthorized"
+        '404':
+          description: "`Dataset` not found"
+          schema:
+            "$ref": "#/definitions/NotFound"
+        '200':
+          description: "`Dataset` response"
+          schema:
+            "$ref": "#/definitions/Report"
+  "/{account}/datsets/{datsets}":
```
Comment (monteverdetico) [inline swagger.yaml]:
```suggestion
  "/{account}/datasets/{dataset}":
```

**Review by monteverdetico — COMMENTED** [review]
Looks good, just one small typo. :)

File: swagger.yaml [hunk 2]
```diff
@@ -231,6 +231,70 @@ paths:
...
+            "$ref": "#/definitions/Unauthorized"
+        '404':
+          description: "`Dataset` not found"
+          schema:
+            "$ref": "#/definitions/NotFound"
+        '200':
+          description: "`Dataset` response"
+          schema:
+            "$ref": "#/definitions/Report"
+  "/{account}/datasets/{datasets}":
```
Comment (monteverdetico) [inline swagger.yaml]:
```suggestion
  "/{account}/datasets/{dataset}":
```

**Review by monteverdetico — APPROVED** [review]
One more tiny typo, and otherwise looks good. :)

**Review by monteverdetico — APPROVED** [review]

**Pull request merged**